smurfs/src/components/Smurf.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Smurf = props => {
+  return <li>{props.smurf.name}</li>
+}
+
+export default Smurf;
```

Comment:
I think this whole single smurf component isn't actually doing anything, you could just take it out if you wanted to make your code more DRY.